src/DotFlyer.Service/AzureDataExplorer/Models/SMSData.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace DotFlyer.Service.AzureDataExplorer.Models;
+
+/// <summary>
+/// Contains the model for SMS data to be ingested into Azure Data Explorer.
+/// </summary>
+public class SMSData : SMSMessage
+{
+    public required string From { get; set; }
+
+    public new required string Tags { get; set; }
```

Comment:
the new keyword here - perhaps we should not tie service model and ADX model.. it is even more prominent with email

Reply:
I understand that... once you have to use the new etc.. the duplication is de-facto happening.. 

we have the option of e.g. automapper etc and keep these concerns separate but not critical